domains/ewan.json

```diff
@@ -0,0 +1,6 @@
+{
+  "owner": {
+    "username": "<ewan-dev>",
+    "discord": "<@meowcuber>"
+  }
+}
```

Comment:
File is invalid mate, please read docs since theres more then just this. (also at owner field remove the <> and add email line which you can leave it blank)